vsphere/README.md

```diff
@@ -1,47 +1,94 @@
-    - [Show Reserved IPs](#show-reserved-ips)
-    - [Show Available IPs in a Segment](#show-available-ips-in-a-segment)
-    - [Generate an `install-config.yaml` Template (and Reserve IPs)](#generate-an-install-configyaml-template-and-reserve-ips)
+    - [Set VCenter Credentials (`new_hub.py` only)](#set-vcenter-credentials-new_hubpy-only)
+  - [Connect to Twingate (`new_hub.py` Only)](#connect-to-twingate-new_hubpy-only)
+- [`vmcdns.py`: View, Reserve, and Release DEVQE VMC DNS Entries](#vmcdnspy-view-reserve-and-release-devqe-vmc-dns-entries)
+  - [Show Reserved IPs](#show-reserved-ips)
+  - [Show Available IPs in a Segment](#show-available-ips-in-a-segment)
+  - [Generate an `install-config.yaml` Template (and Reserve IPs)](#generate-an-install-configyaml-template-and-reserve-ips)
     - [Profit](#profit)
     - [Release IPs](#release-ips)
+- [`new_hub.py`: Provision a VSphere Cluster in DEVQE Without Hive](#new_hubpy-provision-a-vsphere-cluster-in-devqe-without-hive)
+  - [Download `openshift-install`](#download-openshift-install)
+  - [Create the Cluster](#create-the-cluster)
+  - [Destroy the Cluster](#destroy-the-cluster)
 
-## `vmcdns.py`: View, Reserve, and Release DEVQE VMC DNS Entries
-The DEVQE environment has predefined network segments in which static IP addresses can be reserved for use with VSphere clusters.
-This DNS management is done in the [openshift-vmware-cloud-ci AWS account](https://openshift-vmware-cloud-ci.signin.aws.amazon.com/console) (726924432237), for which you should have received credentials during [onboarding](https://docs.google.com/document/d/1cnzKMT-8TGcq5ox_AajGpT-VRwMuYTKIZI-fPNrR8Xg/edit#heading=h.bycjc9mvj2ia).
-
-### One-Time Setup
+## One-Time Setup
 - [Get Python 3](https://www.python.org/downloads/)
 - (Optional) [Set up a virtual environment](https://docs.python.org/3/library/venv.html)
 - Install requirements
   (...in your venv, if using)
   ```
-  curl -O https://raw.githubusercontent.com/openshift-hive/hive-extras/master/vsphere/requirements.txt
-  source /path/to/venv/bin/activate  # If using a venv
-  python -mpip install -r requirements.txt
+  $ curl -O https://raw.githubusercontent.com/openshift-hive/hive-extras/master/vsphere/requirements.txt
+  $ source /path/to/venv/bin/activate  # If using a venv
+  $ python -mpip install -r requirements.txt
   ```
-- Download the executable
+- Download the executable(s)
   ```
-  curl -O https://raw.githubusercontent.com/openshift-hive/hive-extras/master/vsphere/vmcdns.py
-  chmod +x vmcdns.py
-  mv vmcdns.py /somewhere/in/your/PATH
+  for f in vmcdns.py new_hub.py; do
+    curl -O https://raw.githubusercontent.com/openshift-hive/hive-extras/master/vsphere/$f
+    chmod +x $f
+    # Optional; you can also just invoke them with a path
+    mv $f /somewhere/in/your/PATH
+  done
+  ```
+- (`new_hub.py` only) Obtain a [pull secret](https://try.openshift.com/) if you don't already have one.
```

Comment:
Suggestion: Use `https://console.redhat.com/openshift/install/vsphere` instead of `https://try.openshift.com` since this is internal documentation

Reply:
It's pretty unclear where to get the pull-secret from in both places, but maybe https://console.redhat.com/openshift/install/vsphere/user-provisioned is a little closer?

> internal

It's a public repo. But yeah, to be fair, we've got several specific internal things hardcoded in here.